templates/sag-db-oracle/README.md

```diff
@@ -1,14 +1,14 @@
 # Creating webMethods database on Oracle
 
-With this template you can create user, database, and webMethods database schemas on a Oracle database server.
+With this template you can create user, database, and webMethods database schemas on an Oracle database server.
```

Comment:
With this template you can create user, storage tablespaces and webMethods database components on an Oracle database server